packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,7 +5,8 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const { availableCommitTypes } = require( './transform-commit-utils' );
+const transformCommitForCKEditor5Package = require( './transformcommitforckeditor5package' );
```

Comment:
In the future, this function will receive a parameter which allows choosing which "transform" function will be used. 

At this moment we have only one transform, so the parameter is not required.

OTOH, I don't know whether the parameter will have a sense. Maybe you are right now.

ATM I have a "YAGNI" feeling.